src/components/Widgets/DateControl.js

```diff
@@ -0,0 +1,21 @@
+import React, { PropTypes } from 'react';
+import DateTime from 'react-datetime';
+
+export default class DateControl extends React.Component {
+  handleChange = (datetime) => {
+    this.props.onChange(datetime);
+  };
+
+  render() {
+    return (<DateTime
+      timeFormat={false}
+      value={this.props.value || new Date()}
+      onChange={this.handleChange}
+    />);
+  }
+}
+
+DateControl.propTypes = {
+  onChange: PropTypes.func.isRequired,
+  value: PropTypes.object,
```

Comment:
There is a propType for that `PropType.oneOfType` :)

Also, I'd suggest we don't couple with moment.js so tightly. So, I'd say:

```js
PropTypes.oneOfType([
  React.PropTypes.string,
  React.PropTypes.instanceOf(Date)
])
```